crates/bin/prove_block/src/main.rs

```diff
@@ -478,40 +511,74 @@ async fn main() -> Result<(), Box<dyn Error>> {
         );
     }
 
-    let contract_state_commitment_info =
-        CommitmentInfo::create_from_modifications::<CachedRpcStorage, PedersenHash, ContractState>(
-            previous_tree.clone(),
-            None, // TODO: do we have a source for expected?
-            updates,
-            &mut initial_state.ffc,
-        )
-        .await?;
-
-    // Blockifier provides a class hash -> visited PCs map, but we need
-    // the compiled class hash -> visited PCs map.
-    let compiled_class_visited_pcs: HashMap<Felt252, Vec<Felt252>> = blockifier_state
+    let visited_pcs: HashMap<Felt252, Vec<Felt252>> = blockifier_state
         .visited_pcs
         .iter()
         .map(|(class_hash, visited_pcs)| {
-            let class_hash_felt = felt_api2vm(class_hash.0);
-            let compiled_class_hash_felt = class_hash_to_compiled_class_hash.get(&class_hash_felt).unwrap();
-            (*compiled_class_hash_felt, visited_pcs.iter().copied().map(Felt252::from).collect::<Vec<_>>())
+            (felt_api2vm(class_hash.0), visited_pcs.iter().copied().map(Felt252::from).collect::<Vec<_>>())
         })
         .collect();
 
+    log::debug!("class_hash_to_compiled_class_hash ({}):", class_hash_to_compiled_class_hash.len());
+    for (ch, cch) in &class_hash_to_compiled_class_hash {
```

Comment:
put better names to this variables

Reply:
https://github.com/Moonsong-Labs/snos/pull/19/commits/e1f040a66cf661659dc84d0a95ab22162255b4cd